training/run_distillation.py

```diff
@@ -1315,10 +1349,18 @@ def compute_metrics(preds, labels):
         eval_steps = training_args.eval_steps
 
     # 13. Define optimizer, LR scheduler, collator
+    
+    if training_args.freeze_encoder:
+        forbidden_module = [student_model.model.encoder]
+        if training_args.freeze_decoder:
```

Comment:
Proposed code would work for most cases, but would miss the edge case when `--freeze_encoder=False` and `--freeze_decoder=True`

Reply:
Spot on! fixed in [b33e7f8](https://github.com/huggingface/distil-whisper/pull/126/commits/b33e7f8c0b8525dd9dafdb5c13b8adb129d17b84)kljl